LibertyDSNP/example-client
# type changes

Purpose
---------------
Update Example client to SDK V3.0.1
[https://www.pivotaltracker.com/n/projects/2436354](url)

Solution
---------------
change version in package.json, npm install, and fix everything that breaks because of the changes


Change summary
---------------
* changed all bigInt id's to strings where they needed to be
* changed types to the correct versions (ex: HexString => DSNPUserId )
* updated tests that were failing due to changes
* fix any other misc. errors that arose.


Steps to Verify
----------------
1. Read code
2.  Verify all actions are working
 * login
 * post
 * comment
 * edit username
 * switch accounts
 * logout
